src/main/java/org/spongepowered/api/world/Environment.java

```diff
@@ -44,4 +44,52 @@
      */
     String getName();
 
+    /**
+     * Returns whether players can respawn within {@link Environment} after death.
+     *
+     * @return True if players can respawn, false if not
+     */
+    boolean allowsPlayerRespawns();
+
+    /**
+     * Sets whether players in this {@link Environment} can respawn.
+     *
+     * @param allow Whether players can respawn
+     */
+    void setAllowsPlayerRespawns(boolean allow);
+
+    /**
+     * Returns the minimum spawn height for {@link Environment}.
+     *
+     * @return The minimum spawn height
+     */
+    int getMinimumSpawnHeight();
+
+    /**
+     * Returns whether water evaporates for {@link Environment}.
+     *
+     * @return True if water evaporates, false if not
+     */
+    boolean doesWaterEvaporate();
+
+    /**
+     * Sets whether water in this {@link Environment} evaporates.
+     *
+     * @param evaporates Whether water evaporates
+     */
+    void setWaterEvaporates(boolean evaporates);
+
+    /**
+     * Returns if the {@link Environment} has a sky.
```

Comment:
I think this needs more clarification. Does a sky mean the lack of a bedrock ceiling (which the Nether has).